Brain/MarkovBrain/MarkovBrain.cpp

```diff
@@ -176,13 +176,13 @@ void MarkovBrain::update() {
         if (nextNodesConnections[nrInputValues + i] == 0) 
           nextNodes[nrInputValues + i] = Random::getDouble(
               randomizeUnconnectedOutputsMin, randomizeUnconnectedOutputsMax);
-      break;
-    default:
-      std::cout
-          << "  ERROR! BRAIN_MARKOV_ADVANCED::randomizeUnconnectedOutputsType "
-             "is invalid. current value: "
-          << randomizeUnconnectedOutputsType << std::endl;
-      exit(1);
+      //break;
+    //default:
+      //std::cout
+      //    << "  ERROR! BRAIN_MARKOV_ADVANCED::randomizeUnconnectedOutputsType "
+      //       "is invalid. current value: "
+      //    << randomizeUnconnectedOutputsType << std::endl;
+      //exit(1);
```

Comment:
Do this commented code really need to remain here?

Reply:
we may want to add other randomizeUnconnectedOutputTypes, if so, this will save time.

Reply:
Sounds good.